### PR TITLE
Symofny / Sylius / Framework

### PR DESCRIPTION
**Pull request by mkalkowski83**
Changelog:
1. Add information about autowire and autoconfiguration
2. Add information about FQDN in taged services

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,8 @@ $flavors = [
 8. Creating a CLI Command using Symfony Console Component should follow the following rules:
     - `execute` method should have `int` as a return type. For the **successful** run, the command should return `0`. For any errors during execution the return can be `1` or any different *error code number*.
 9. In Sylius plugins, use traits for customizing models and use them inside your `tests/Application/src` for testing. This way we avoid handling reference conflicts in the final app.
+10. We doesn't use `autowire` and `autoconfigure`.
```
Comment (RadnoK) [inline README.md]: Should be `We don't` 😉 . Also, I would write it this way:

>We don't use either `autowire` nor `autoconfigure` Symfony options as it is a very "magic" way of defining services. We prefer to always manually define services and inject proper arguments into them to have better control of our Container.

Comment (mkalkowski83) [reply README.md]: Done ;)

File: README.md [hunk 2]
```diff
@@ -128,6 +128,8 @@ $flavors = [
 8. Creating a CLI Command using Symfony Console Component should follow the following rules:
     - `execute` method should have `int` as a return type. For the **successful** run, the command should return `0`. For any errors during execution the return can be `1` or any different *error code number*.
 9. In Sylius plugins, use traits for customizing models and use them inside your `tests/Application/src` for testing. This way we avoid handling reference conflicts in the final app.
+10. We don't use either autowire nor autoconfigure Symfony options as it is a very "magic" way of defining services. We prefer to always manually define services and inject proper arguments into them to have better control of our Container.
+11. If some of service definition is taged don't use FQDN as service id.
```
Comment (RadnoK) [inline README.md]: Oh, also this. Should be `FQCN` - Fully Qualified *Class* Name, not *Domain* 😄

Comment (mkalkowski83) [reply README.md]: Done

Comment (bitbager) [reply README.md]: `We always prefer to manually define...*`
`If some of the service definition is tagged, don't use FQCN ( Fully Qualified Class Name) as the service id.`*

Comment (mkalkowski83) [reply README.md]: better?

File: README.md [hunk 3]
```diff
@@ -128,6 +128,8 @@ $flavors = [
 8. Creating a CLI Command using Symfony Console Component should follow the following rules:
     - `execute` method should have `int` as a return type. For the **successful** run, the command should return `0`. For any errors during execution the return can be `1` or any different *error code number*.
 9. In Sylius plugins, use traits for customizing models and use them inside your `tests/Application/src` for testing. This way we avoid handling reference conflicts in the final app.
+10. We don't use either autowire nor autoconfigure Symfony options as it is a very "magic" way of defining services. We always prefer to manually define services and inject proper arguments into them to have better control of our Container.
+11. If some of the service definition is tagged, don't use FQCN (Fully Qualified Class Name) as the service id.*
```
Comment (RadnoK) [inline README.md]: Redundant `*` on the line ending.

Comment (mkalkowski83) [reply README.md]: done

**Comment by bitbager** [comment]
Thank you, Michał!

**Pull request merged**